packages/editor/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.1 (Unreleased)
+
+- Add animated logo to preview interstitial screen.
```

Comment:
I'd just add a title for this. Hesitating between `## New Features` which means we'd probably have to pick `5.1.0` as a version of just `## Polish` where we'd keep `5.0.1`